craft_cli/dispatcher.py

```diff
@@ -39,6 +39,9 @@ class CommandGroup(NamedTuple):
     commands: Sequence[type[BaseCommand]]
     """A list of the commands belonging in this group."""
 
+    ordered: bool = False
+    """Whether this command group is already in the correct group (defaults to False)."""
```

Comment:
```suggestion
    """Whether the commands in this group are already in the correct order (defaults to False)."""
```

Reply:
That's what I get for typing a docstring during huddle